ani-cli

```diff
@@ -371,12 +397,15 @@ while :; do
 			;;
 
 		q)
+			#first kill the background process checking for playback progress then break
+			jobtokill=$(jobs -l | grep "Running" | grep -o "[[:digit:]]\{3,10\}")
```

Comment:
there is no `jobs` command in posix shell so we would need to have a pidfile for the running bg process. That we read a kill later on

Reply:
Damn. My linter probably didn't catch this since it's inside the $()